projects/core/src/checkout/store/effects/address-verification.effect.ts

```diff
@@ -18,10 +16,10 @@ export class AddressVerificationEffect {
     map((action: any) => action.payload),
```

Comment:
maybe add a type here

Reply:
done by adding a type to `ofType` operator above. if you hover of `action` now you'll see a type.